core/retail-basket.tab.info

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "retail-basket",
+    "description": "This is a transnational data set which contains all the transactions occurring between 01/12/2010 and 09/12/2011 for a UK-based and registered non-store online retail.The company mainly sells unique all-occasion gifts. Many customers of the company are wholesalers.",
+    "title": "Online Retail",
+    "tags": ["marketing"],
+    "target": "",
+    "version": "1.0",
+    "collection": "UCI",
+    "instances": 914,
+    "missing": false,
+    "variables": 50,
+    "year": "2012",
+    "references": [
+        "Daqing Chen, Sai Liang Sain, and Kun Guo (2012) Data mining for the online retail industry: A case study of RFM model-based customer segmentation using data mining. Journal of Database Marketing and Customer Strategy Management 19(3), pp. 197."
+    ],
+    "size": 665,
```

Comment:
Not sure what this is, but it is not the size of the file in bytes.
Please delete this field - it is added automatically later.